dockerfiles/custom_images.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
missing copyright headers
Also I'm not sure that we require another top level script
IMHO it should be integrated in build.include